Gopkg.lock

```diff
@@ -272,8 +266,8 @@
   branch = "handle-relative-nextlinks"
```

Comment:
If we own it, can't this be master?

Reply:
Yeah I suppose. I'll change this and then merge.